README.md

```diff
@@ -32,14 +32,15 @@ Any argument is valid, but most are ignored. These are the expected ones (all ot
 - `--exclude PATH` Can be added multiple times. Will ignore listed files in vue source directory (paths should be relative to it)
 - `--icon PATH` Icon file to add to the zipped XUI and pointed to in the metadata.
 
-In addition, any other argument that's prepended with `met` gets added to the metadata. Important examples and how the content library or CloudBolt uses them(note the underscores in the keys):
+In addition, any other argument that's prepended with `met` gets added to the metadata. Important examples and how the content library or CloudBolt uses them (note the underscores in the keys):
 
 - `--met_description "TEXT"` Content Library Description. Make sure to quote arguments that have spaces in them.
 - `--met_label LABEL` Human-readable version of the name.
 - `--met_version VERSION` XUI version. By default, this is read from the package.json's `version` field.
 - `--met_minimum_version_required VERSION` Minimum CloudBolt version number for the XUI
 - `--met_maximum_version_required VERSION` Maximum CloudBolt version number for the XUI
 - `--met_last_updated YYYY-MM-DD` Defaults to today unless specified
+- `--met_inbound_web_hook_dependencies` Array of inbound webhooks (iwh) files that are included with this XUI
```

Comment:
Wouldn't the inbound webhook exist as a directory? Is the user expected to list the contents of the directory in this argument?

Also, is it just me, or does this sort of make it sound like it's expecting files with a .iwh extension?

Reply:
So I'm just basing it off how it downloads from the HUI export manager when I add the IWHs as dependencies to the applet through the CMP and [Orvis's suggestions in my other PR](https://github.com/CloudBoltSoftware/applet-s3-bucket-browser/pull/6#pullrequestreview-1768265906) to drop the files in `xui/src` (though that may have been under the assumption that it would automatically work, and given the code changes needed could be its own folder now, I guess)
![Screenshot 2023-12-11 at 2 44 05 PM](https://github.com/CloudBoltSoftware/xui-packager/assets/105660893/a00d65f5-a234-4701-9806-8c04b0812990)

Reply:
(also good point, my wording in the README is not great, lemme adjust)

Reply:
Alyssa asked me to chime in.

This story is relevant here: https://cloudbolt.atlassian.net/browse/CMP-1127. It has two goals:
 - Automatically generate Inbound Web Hook content from source code.
 - Create a strategy to automatically bundle the :point_up: generated Inbound Web Hook into the automatically bundled applet.
This PR addresses the second goal, not the first. So I think it's fine that the user is expected to list all the iwh zip files in the metadata. It's just using the general-purpose nature of any `met_` prefixed config.

I'd suggest a small change to the implementation to get to that goal in this PR. Trade-off wise, I'd prefer this tool remain pretty general purpose and modular over magic/super easy to use. Bundling IWHs is a specific use case for a more general-goal: how can we bundle files into the root of the bundled zip? If we solve the general case, we can use this tool without alteration the next time something like this comes up.

I'd suggest adding an additional arg, something like `--additional-files-dir` that (if supplied) adds any files in in it into the main zip file. Then the use that flag pointing to `xui/additional_files` or something that includes all the iwhs. And use the `met_inbound_web_hook_dependencies` to add that list to the metadata.

How does that sound?